app/models/target.rb

```diff
@@ -0,0 +1,32 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id               :bigint           not null, primary key
+#  location_id      :bigint
+#  sector_id        :bigint
+#  target_scope_id  :bigint
+#  ghg_target       :boolean          default(FALSE), not null
+#  single_year      :boolean          default(FALSE), not null
+#  description      :text
+#  year             :integer
+#  base_year_period :string
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#
+
+class Target < ApplicationRecord
+  self.inheritance_column = nil
```

Comment:
maybe to early to ask, but why do we need this here? we don't have a `type` column here.

Reply:
I had a type column, and removed it for now. I forgot to remove this line though.